pkg/doc/did/doc.go

```diff
@@ -53,6 +56,294 @@ const (
 	jsonldPublicKeyjwk    = "publicKeyJwk"
 )
 
+const didV1Context = `
```

Comment:
can we move contexts to a separate file?

Reply:
Sure, @troyronda . What's the best place for it? At the same level as the source files?

Reply:
sure for now.

Reply:
Added `context` folder.

Reply:
Corrected - JSON-LD contexts definition is moved into separate `contexts.go`.